test/role.test.js

```diff
@@ -431,6 +431,179 @@ describe('role model', function() {
     });
   });
 
+  it('should support owner role resolver only with flag true', function(done) {
+    const Message = app.registry.createModel('message',
+    { content: String },
+    {
+      relations: {
+        user: {
+          type: 'belongsTo',
+          model: 'User',
+          foreignKey: 'userId',
+        },
+        sender: {
+          type: 'belongsTo',
+          model: 'User',
+          foreignKey: 'senderId',
+        },
+      },
+      ownerRelations: true,
+    });
+
+    app.model(Message, {dataSource: 'db'});
+
+    const password = 'pass';
```

Comment:
I am afraid we cannot use ES6 in this repository yet because of https://github.com/strongloop/loopback/issues/3003 :(